tests/project/goost/scene/gui/test_grid_rect.gd

```diff
@@ -2,6 +2,7 @@ extends "res://addons/gut/test.gd"
 
 
 func test_snap():
+	Input.use_accumulated_input = false
```

Comment:
Input was being buffered so the test would forever wait. Looks like this was changed in 3.5? https://github.com/bitwes/Gut/wiki/Mock-Input#using-input-as-a-receiver

Preferrably we'd use GUT's InputSender but that would require rewriting the unit tests within GUT rather than the test scene script.